### DISCUSSION TITLE
Added LICENSE.md

### DISCUSSION THREAD

**Discussion by ipspace** [ipspace/netlab#205] [General]
@ssasso @petercrocker @jbemmel:

Added MIT license to the repository (in LICENSE.md). While I like https://github.com/dtf0/wtfpl, it might be "not safe for work" in some countries.

Hope you're OK with that.

**Comment by petercrocker** [comment]
I’d not seen that license before, similar to an unlicense but with clearer language ;) https://en.wikipedia.org/wiki/Unlicense

I’m good with MIT, but it’s mostly your code, your call!

**Comment by ssasso** [comment]
> I’m good with MIT, but it’s mostly your code, your call!

Same for me ;)

**Comment by ipspace** [comment]
Added LICENSE.md a long while ago
